tests/e2e/playbooks/templates/user-data.j2

```diff
@@ -0,0 +1,14 @@
+#cloud-config
+users:
+  - default
+  - name: {{ ssh_user }}
+    groups: wheel
+    sudo: ALL=(ALL) NOPASSWD:ALL
+    lock_passwd: true
+    ssh_authorized_keys:
+      - {{ lookup('ansible.builtin.file', ssh_key_pub) }}
+power_state:
```

Comment:
Interesting...wasn't aware of this one. COuld use a comment I think...something like:
`# Shut down after the testing script has executed` or so?

Reply:
Comment added.